### DISCUSSION TITLE
Java 11 on GitHub Actions?

### DISCUSSION THREAD

**Discussion by sblantipodi** [community/community#26929] [Actions]
<p>As title.<br>
How can I use JDK 11 on GitHub Actions?</p>
<p>Thanks</p>

**Comment by weide-zhou** [comment, marked as answer]
<p>Hi <a class="mention" href="/sblantipodi">@sblantipodi</a>,</p>
<p>Do you mean using JDK 11 in workflow?  You can use ‘actions/setup-java@v1’ to setup jdk. Code sample as below:</p>
<pre><code class="lang-auto">      - uses: actions/setup-java@v1
        with:
          java-version: '11'
</code></pre>
<p>More details please check the action usage here: <a href="https://github.com/actions/setup-java#setup-java" rel="nofollow noopener">https://github.com/actions/setup-java#setup-java</a>.</p>
<p>Thanks</p>

**Comment by Sandwich872** [comment]
how can u unlock a I phone 11 that u don't  know password or email
